src/main/scala/nl.knaw.dans.easy.springfield/ListFiles.scala

```diff
@@ -20,14 +20,14 @@ import scala.xml.Elem
 
 trait ListFiles {
 
-  def listFiles(parent: Elem, presentationId: Int): Seq[(Int, String,String, Boolean)] = {
+  def listFiles(parent: Elem, presentationId: String): Seq[(String, String,String, Boolean)] = {
     for {
       presentation <- (parent \ "user" \ "presentation").theSeq
-      if (presentation \@ "id") == presentationId.toString
+      if (presentation \@ "id") == presentationId.replaceAll(".*/","")
```

Comment:
what do you expect this presentationId to be? I'm imagining it is something like `easy-dataset:415`

so you will need to use both `<presentation>` blocks, one for the @id and one for the videoplaylist